webapp/portlet/src/main/webapp/vue-apps/common/initComponents.js

```diff
@@ -196,7 +196,8 @@ Vue.directive('cacheable', {
 Vue.directive('draggable', {
   bind(el, binding) {
     let counter = 0;
-    const enabled = binding?.value;
+    const objectType = binding?.value;
+    const enabled = eXo.env.portal.editorAttachImageEnabled && objectType?.length && eXo.env.portal.attachmentObjectTypes?.indexOf(objectType) >= 0;
```

Comment:
This directive is dissociated from image attachments, so please don't embed 'Image attachment' feature properties into it.

```suggestion
    const enabled = binding?.value;
```

Reply:
fixed!